cmd/osbuild-image-tests/main_test.go

```diff
@@ -179,6 +180,15 @@ func testSSH(t *testing.T, address string, privateKey string, ns *boot.NetNS) {
 }
 
 func testBootUsingQemu(t *testing.T, imagePath string) {
+	if *failLocalBoot {
+		t.Fatal("-fail-local-boot specified. Check missing cloud credentials!")
+	}
+
+	bootWithQemu(t, imagePath)
+}
+
+// will not fail even if -fail-local-boot is specified
+func bootWithQemu(t *testing.T, imagePath string) {
```

Comment:
I think this function name is kinda confusing. I suggest moving the condition to `testBootUsingVMware` and the other cloud-specific functions. That makes more sense to me.